v2/cmd/asoctl/pkg/importresources/resource_importer.go

```diff
@@ -37,9 +37,8 @@ type ResourceImporterOptions struct {
 }
 
 type ImportResourceResult struct {
-	resource ImportedResource
-	pending  []ImportableResource
-	err      error
```

Comment:
Now using a separate channel to return failures, which gives this struct a firmer purpose.